src/librustc/hir/intravisit.rs

```diff
@@ -500,7 +500,7 @@ pub fn walk_item<'v, V: Visitor<'v>>(visitor: &mut V, item: &'v Item) {
         ItemKind::GlobalAsm(_) => {
             visitor.visit_id(item.hir_id);
         }
-        ItemKind::Ty(ref ty, ref generics) => {
+        ItemKind::TyAlias(ref ty, ref generics) => {
```

Comment:
Better yet just `::TypeAlias`?

Reply:
I think `Ty` is slightly more common in the codebase. We should make it consistent, though.